src/filter-context/filter-context.js

```diff
@@ -4,7 +4,7 @@ import { reducerFunc } from '../reducer-func/reducer-func'
 const FilterContext=createContext()
 
 const FilterProvider=({children})=>{
-    const [state, dispatch] = useReducer(reducerFunc, {sorting:null})
+    const [state, dispatch] = useReducer(reducerFunc, {sorting:null,isInStock:false,fastDelivery:false,priceRange:50000,rating:5,category:{ios:false,android:false}})
```

Comment:
you can create an object called `initialState` and extract out these properties to make the code more clean. Although, it is fine too.

Reply:
ok will do it